code/game/gamemodes/cult/structures/desk.dm

```diff
@@ -0,0 +1,4 @@
+/obj/structure/cult/tome
+	name = "arcanaeum desk"
+	desc = "A desk covered in arcane manuscripts and tomes in unknown languages. Looking at the text makes your skin crawl."
```

Comment:
Maybe the desc should depend on if you're a cultist or not? Seems weird for it to make cultists' skin crawl.